MantidQt/CustomInterfaces/src/Indirect/IndirectDiffractionReduction.cpp

```diff
@@ -487,6 +487,10 @@ void IndirectDiffractionReduction::instrumentSelected(
   double specMin = instrument->getNumberParameter("spectra-min")[0];
   double specMax = instrument->getNumberParameter("spectra-max")[0];
 
+  if (instrumentName == "OSIRIS" && reflectionName == "diffspec") {
+	  specMax = 15;
```

Comment:
Yes that's a good idea. I'll find a more sensible place to put it in the instrument parameter file